core/soong_plugin.go

```diff
@@ -0,0 +1,117 @@
+// +build ignore
```

Comment:
Can we put this after the copyright declaration please. I believe it just needs to be before the `package` declaration. You might even put it after the comment describing why we have it.

Reply:
I assume microfactory is ignoring the build tags and doing exactly what the Android.bp says. So can we use `+build soong` here? That way we could do some testing (by specifying --tags soong)

Reply:
I tried putting it after the copyright and comment first, unfortunately it doesn't work. Also no luck just after the copyright. In both cases I've tried various amounts of surrounding whitespace.

Microfactory is indeed ignoring the tags, since it's overridden by `Android.bp`.

`+build soong` does work, and it means I can cause e.g. `go vet` to re-include the file with `go vet --tags soong github.com/ARM-software/bob-build/core`.